newsletter-2021-02.html

```diff
@@ -0,0 +1,92 @@
+<h1>tedana February 2021 Update</h1>
+Since our late December newsletter, we have created a new stable release available on PyPI!
+You can upgrade your local installation to the latest 0.0.9 version by running <xmp>pip install -U tedana</xmp>.
+We encourage everyone to go and check it out, as it includes many new bug fixes, improvements, and breaking changes.
+
+<h2>New Contributors</h2>
+We are excited to welcome three new contributors since our last newsletter: M. Versluis, Z. Ahmed, and A. Khojandi!
+Thanks to all of our contributors for their continued input and help in creating tedana.
+The community has now grown to 20 contributors, and we always look forward to seeing more new faces!
+If you’re not sure where to start, please feel free to open an issue or join our next
+<a href="https://tedana.readthedocs.io/en/latest/developing.html?highlight=developer%20call#monthly-developer-calls">monthly developer's call</a>.
+
+<h2>Governance Updates</h2>
+We have recently overhauled our governance structure, moving from a Benevolent-Dictator-for-Life (BDFL) model to a Steering committee structure.
```

Comment:
```suggestion
We have recently overhauled our governance structure, moving from a Benevolent-Dictator-for-Life (BDFL) model to a steering committee structure.
```

Or "Steering Committee", I guess. I just don't like "Steering committee".